webhookValidationConfig.go

```diff
@@ -51,7 +51,7 @@ type OptionsConfig struct {
 }
 
 // BuildURLChecker translates the configuration into url Checker to be run on the webhook.
-func (config *ValidatorConfig) BuildURLChecker() (*urlegit.Checker, error) {
```

Comment:
I may move this to another pr, but this doesn't have to be a member (pointer) of ValidatorConfig. This will simplify the mental model.

Reply:
This change seems to add no value, why is it here?

Reply:
It's only here because I accidentally commit it with the other changes. Fair, I'll trash them 🗑️